app/components/EventDetails.tsx

```diff
@@ -5,10 +5,11 @@ import { AnchorWithArrow } from "./AnchorWithArrow";
 import * as styles from "./EventDetails.css";
 
 export interface EventDetailsProps {
```

Comment:
(I'll say, it is _very nice_ to see someone with your expertise and experience resembling my own coding style. It boosts my own ego lol)